portfolio/src/main/webapp/index.html

```diff
@@ -39,7 +39,11 @@ <h1>Tal's Portfolio</h1>
 
 
       <form id='userOnly' method="POST" action="/data">
-        <p>Post a comment here!</p>
+        <p>Post a comment here as:</p>
+        
+        <!--stores email from user server in this form to send to data server-->
+        <p name = "email" id ="poster"></p>
```

Comment:
This would not send the email to `/data` since `email` here is just a display element `<p>`.  Forms would only send `<input>` elements to the server.  Try instead:
`<input name = "email" id ="poster" value="" />`

Then use CSS to make it look like a read-only input box so users wouldn't update.